runtime/interpreter/encoding_test.go

```diff
@@ -640,12 +757,6 @@ func TestEncodeDecodeComposite(t *testing.T) {
 					0x6a,
 					0x54, 0x65, 0x73, 0x74, 0x53, 0x74, 0x72, 0x75,
 					0x63, 0x74,
-					// key 1
```

Comment:
As you can see, the composite no longer stores key 1 in the CBOR map, which was the type identifier, but now stores key 4, the qualified identifier (see next hunk)